src/frontend/components/injector-tree/injector-tree.ts

```diff
@@ -62,6 +68,8 @@ export class InjectorTree implements OnChanges {
 
     this.parentHierarchy = [{ name: 'root', dependencies: [] }].concat(mainHierarchy).concat([this.selectedNode]);
 
+    this.breadcrumbs = this.parentHierarchy.slice(1, this.parentHierarchy.length);
```

Comment:
@rajinder-yadav In this case I do need two separate versions of the array. Just one without the first ("root") element. Removing the second argument to `slice` is good advice though.